assistedInstaller.py

```diff
@@ -143,6 +143,6 @@ def get_ai_cluster_info(self, cluster_name: str) -> AssistedClientClusterInfo:
         if "id" not in cluster_info:
             logger.error(f"ID is missing in cluster info for cluster {cluster_name}")
             sys.exit(-1)
-        if "api_ip" not in cluster_info:
+        if "api_vip" not in cluster_info:
```

Comment:
This is correct:
api_vip: 192.168.122.99                                                                                                                                                                       
api_vip_dns_name: 192.168.122.99                                                                                                                                                              
api_vips: [{'cluster_id': '421b7fb0-196a-4558-b793-c37273a24ee0', 'ip': '192.168.122.99', 'verification': 'succeeded'}] 
....
id: 421b7fb0-196a-4558-b793-c37273a24ee0